### PR TITLE
Database related changes.

### PR DESCRIPTION
**Pull request by mooskagh**

### CODE REVIEW

File: backend/tournaments/migrations/0001_initial.py [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+            name='TournamentPlayer',
+            fields=[
+                ('id', models.AutoField(verbose_name='ID', serialize=False, primary_key=True, auto_created=True)),
+                ('fixed_rating', models.IntegerField(help_text="Fixed raiting. If NULL, it means that the bot haven't filled the rating yet", null=True, blank=True)),
+                ('member', models.ForeignKey(to='tournaments.Tournament')),
+                ('tournament', models.ForeignKey(to='tournaments.Tournament', related_name='players')),
+            ],
+        ),
+        migrations.AddField(
+            model_name='round',
```
Comment (michal-k) [inline backend/tournaments/migrations/0001_initial.py:95]: This is hard to follow. Can we somehow make the whole table defined in a single place without AddField?

File: backend/tournaments/models.py [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+  tds = models.ManyToManyField(Member)
+
+
+class BucketPlayer(models.Model):
+  bucket = models.ForeignKey(Bucket, related_name='players',
+    help_text='Player of which bucket.')
+
+  member = models.ForeignKey(Member, help_text='Which user.')
+
+  # TODO(crem): Do we need `status` here?
```
Comment (michal-k) [inline backend/tournaments/models.py:77]: Not necessarily. If someone withdraws or is banned, we can immediately set all his games in his future rounds.

File: backend/tournaments/migrations/0001_initial.py (hunk 1 above)
Comment (mooskagh) [inline backend/tournaments/migrations/0001_initial.py:95]: Sorry, that file was autogenerated from model.py and actually is not intended for review.
The one needs to be reviewed is model.py

(It's needed for django to update schema automatically on deployment. And when later database schema is changed again, there will be another 0002_????.py script to update schema in prod)

File: backend/tournaments/models.py (hunk 2 above)
Comment (mooskagh) [inline backend/tournaments/models.py:77]: Ack.

File: backend/tournaments/migrations/0001_initial.py (hunk 1 above)
Comment (michal-k) [inline backend/tournaments/migrations/0001_initial.py:95]: And this file need to be checked-in, or will it be re-autogenerated each time we run the backend?
